common/leaderelection/leaderelection.go

```diff
@@ -40,7 +40,7 @@ type LeaderCallbacks struct {
 
 func NewElector(ctx context.Context, eventBusConfig eventbusv1alpha1.BusConfig, clusterName string, clusterSize int, namespace string, leasename string, hostname string) (Elector, error) {
 	switch {
-	case strings.ToLower(os.Getenv(common.EnvVarLeaderElection)) == "k8s":
+	case eventBusConfig.Kafka != nil || strings.ToLower(os.Getenv(common.EnvVarLeaderElection)) == "k8s":
 		return newKubernetesElector(namespace, leasename, hostname)
```

Comment:
So, if Kafka is defined we still do leader election here? (even though it's "master/master")

Reply:
We still need the leader election for the eventsource. For example if an eventsource uses the calendar we need active/passive so that both pods aren't emitting calendar events at the same time.

Reply:
Got it. And I see it's not being called on the Sensor side.